src/OutputValidation.hpp

```diff
@@ -3,12 +3,15 @@
 struct ExecResult;
 namespace Frozen { struct DagNode; }
 
-enum ValidationResult
+namespace ValidationResult
 {
-    Pass,
-    SwallowStdout,
-    UnexpectedConsoleOutputFail,
-    UnwrittenOutputFileFail
-};
+    enum Enum
```

Comment:
The naming is a bit unusual here. How about `Validation::Result` ?

Reply:
Or maybe this is a convention in this codebase, not worth changing?

Reply:
@lucasmeijer  what's the reason for this change?

Reply:
there's a convention of putting enum values inside a struct named Enum in a namespace.  this specific enum was not following that convention, so I just moved the existing values into an enum struct.